cpg_workflows/jobs/joint_genotyping.py

```diff
@@ -509,3 +564,50 @@ def add_make_sitesonly_job(
     if output_vcf_path:
         b.write_output(j.output_vcf, str(output_vcf_path).replace('.vcf.gz', ''))
     return j, j.output_vcf
+
+def add_split_multiallelics_job(
+    b: hb.Batch,
+    input_vcf: hb.ResourceGroup,
+    output_vcf_path: Path | None = None,
+    job_attrs: dict | None = None,
+) -> tuple[Job | None, hb.ResourceGroup]:
+    """
+    Create multiallelic-split VCF
+
+    Returns: a Job object with a single output j.split_vcf of type ResourceGroup
+    """
+    if output_vcf_path and can_reuse(output_vcf_path):
+        return None, b.read_input_group(
+            **{
+                'vcf.gz': str(output_vcf_path),
+                'vcf.gz.tbi': str(output_vcf_path) + '.tbi',
+            }
+        )
+
+    j = b.new_job('MakeSitesOnlyVcf', job_attrs or {} | {'tool': 'bcftools norm'})
+    j.image(image_path('bcftools'))
+    # this is only for a single fragment, so expect it to be small
+    j.storage(get_config()['workflow'].get('splitting_gb', '10Gi'))
+
+    j.declare_resource_group(
+        output_vcf={'vcf.gz': '{root}.vcf.gz', 'vcf.gz.tbi': '{root}.vcf.gz.tbi'}
+    )
+
+    # bcftools norm splits multiallelic variants
+    # -m -any: split all multiallelic variant types
+    # -N: don't left-align indels (NO Normalisation)
```

Comment:
I would be keen to discuss this at some point. I know it would require some further rejig of our pipelines, but it still feels like normalisation would make sense for us.